library/src/main/java/com/airbnb/android/airmapview/WebAirMapViewBuilder.java

```diff
@@ -13,13 +15,38 @@
   }
 
   /**
+   * Set Mapbox Access Token for Web map
+   * @link https://www.mapbox.com/help/define-access-token/
+   * @param mapboxAccessToken Mapbox Access Token
+   * @return WebAirMapViewBuilder
+   */
+  public WebAirMapViewBuilder setMapboxAccessToken(String mapboxAccessToken) {
+    this.mapboxAccessToken = mapboxAccessToken;
+    return this;
+  }
+
+  /**
+   * Set Mapbox Map Id
+   * @link https://www.mapbox.com/help/define-map-id/
+   * @param mapboxMapId
+   * @return WebAirMapViewBuilder
+   */
+  public WebAirMapViewBuilder setMapboxMapId(String mapboxMapId) {
+    this.mapboxMapId = mapboxMapId;
+    return this;
+  }
+
+  /**
    * Build the map fragment with the requested options
    *
    * @return The {@link WebViewMapFragment} map fragment.
    */
   @Override public WebViewMapFragment build() {
     if (options == null) {
-      options = new GoogleWebMapType();
+      options = new MapboxWebMapType(mapboxAccessToken, mapboxMapId);
+    }
+    if (options instanceof MapboxWebMapType) {
```

Comment:
add logic here to not return a `MapboxWebViewMapFragment` if we don't have a mapbox ID and token set